rakelib/rswag.rake

```diff
@@ -13,17 +13,21 @@ namespace :rswag do
 
   namespace :appeals_api do
     desc 'Generate rswag docs and schemas for appeals_api'
-    task run: %i[write_json_schema rswag]
+    task run: %i[prod]
 
-    task write_json_schema: :environment do
-      # require AppealsApi::Engine.root.join('spec', 'support', 'rswag_config.rb')
-      # AppealsApi::RswagConfig.new.write_schema
+    task prod: :environment do
+      ENV['PATTERN'] = 'modules/appeals_api/spec/docs/'
+      ENV['RAILS_MODULE'] = 'appeals_api'
+      ENV['SWAGGER_DRY_RUN'] = '0'
+      Rake::Task['rswag:specs:swaggerize'].invoke
     end
 
-    task rswag: :environment do
-      ENV['PATTERN'] = 'modules/appeals_api/spec/docs'
+    task dev: :environment do
+      ENV['PATTERN'] = 'modules/appeals_api/spec/docs/'
+      ENV['RSWAG_ENV'] = 'dev'
       ENV['RAILS_MODULE'] = 'appeals_api'
       ENV['SWAGGER_DRY_RUN'] = '0'
+      ENV['WIP_DOCS_ENABLED'] = %w[supplemental_claims nod_v2_claimant].join(',')
```

Comment:
Again to be certain, supplemental_claims is here only for demonstration?

Reply:
Also yes.